File: modelMain/src/main/java/com/wynne/knowledge/tree/book/AnimationPathActivity.java
package com.wynne.knowledge.tree.book;

import android.graphics.drawable.Animatable;
import android.support.graphics.drawable.AnimatedVectorDrawableCompat;
import android.widget.ImageView;

import com.wynne.knowledge.tree.R;
import com.wynne.knowledge.tree.base.BaseActivity;

/**
 * @author Wynne
 * @date 2018/10/25
 */

public class AnimationPathActivity extends BaseActivity {
    @Override
    public void initView() {
        ImageView imageView = findViewById(R.id.iv_vector);
        AnimatedVectorDrawableCompat vectorDrawable = AnimatedVectorDrawableCompat.create(AnimationPathActivity.this, R.drawable.line_animated_svg);
        imageView.setImageDrawable(vectorDrawable);
        ((Animatable) imageView.getDrawable()).start();
    }

    @Override
    public int getLayoutId() {
        return R.layout.animation_path_activity;
    }
}

File: modelMark/src/main/java/com/wynne/knowledge/mark/activity/AnimationPathActivity.kt
package com.wynne.knowledge.mark.activity

import android.support.graphics.drawable.AnimatedVectorDrawableCompat
import com.wynne.knowledge.base.base.BaseActivity
import com.wynne.knowledge.mark.R
import kotlinx.android.synthetic.main.animation_path_activity.*

class AnimationPathActivity : BaseActivity() {
    override fun getLayoutId(): Int = R.layout.animation_path_activity

    override fun initView() {
        val vector = AnimatedVectorDrawableCompat.create(this@AnimationPathActivity, R.drawable.line_animated_svg)
        ivVector.setImageDrawable(vector)
        vector?.start()
    }
}